### PR TITLE
update to dxp 7.2.10 in sdk installer

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

File: integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java [hunk 1]
```diff
@@ -87,7 +87,8 @@ public void testNewMavenLiferayWorkspaceInitBundle() throws Exception {
 		NewLiferayWorkspaceOp op = NewLiferayWorkspaceOp.TYPE.instantiate();
 
 		String defaultBundleUrl =
-			"https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip";
+			"https://releases-cdn.liferay.com/portal/7.2.0-ga1/liferay-ce-portal-tomcat-7.2.0-ga1-" +
```
Comment (jtydhr88) [inline integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java]: @yanan let's use WorkspaceConstants.BUNDLE_URL_CE_7_2 instead.

File: integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java [hunk 2]
```diff
@@ -169,7 +210,8 @@ public void testNewMavenLiferayWorkspaceSetUrl() throws Exception {
 		NewLiferayWorkspaceOp op = NewLiferayWorkspaceOp.TYPE.instantiate();
 
 		String bundleUrl =
```
Comment (jtydhr88) [inline integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java]: @yanan let's use WorkspaceConstants.BUNDLE_URL_CE_7_2 instead.

File: integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java [hunk 3]
```diff
@@ -123,6 +124,46 @@ public void testNewMavenLiferayWorkspaceInitBundle() throws Exception {
 		deleteProject(workspace.getName());
 	}
 
+	@Test
+	public void testNewMavenLiferayWorkspaceOpWithBundle70() throws Exception {
+		NewLiferayWorkspaceOp op = NewLiferayWorkspaceOp.TYPE.instantiate();
+
+		String default70BundleUrl =
```
Comment (jtydhr88) [inline integration-tests/tools/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/NewLiferayWorkspaceMavenTests.java]: @yanan let's use WorkspaceConstants.BUNDLE_URL_CE_7_0 instead.

**Comment by yanan** [comment]
Ok, will resend.

**Pull request closed without merging**